src/components/CompareResults/RevisionRow.tsx

```diff
@@ -329,46 +414,7 @@ function RevisionRow(props: RevisionRowProps) {
             </div>
           </>
         )}
-        <div className='status cell' role='cell'>
-          <Box
-            sx={{
-              bgcolor: improvement
-                ? 'status.improvement'
-                : regression
-                  ? 'status.regression'
-                  : 'none',
-            }}
-            className={`status-hint ${determineStatusHintClass(
-              !!improvement,
-              !!regression,
-            )}`}
-          >
-            {improvement ? <ThumbUpIcon color='success' /> : null}
-            {regression ? <ThumbDownIcon color='error' /> : null}
-            {testVersion === MANN_WHITNEY_U
-              ? capitalize(
-                  (result as MannWhitneyResultsItem).direction_of_change ?? '',
-                )
-              : determineStatus(!!improvement, !!regression)}
-          </Box>
-        </div>
-        <div className='delta cell' role='cell'>
-          {' '}
-          {testVersion === MANN_WHITNEY_U
-            ? (result as MannWhitneyResultsItem).cliffs_delta &&
-              `${(result as MannWhitneyResultsItem).cliffs_delta}`
-            : ` ${deltaPercent} % `}
-        </div>
-        {testVersion === MANN_WHITNEY_U ? (
-          <div className='confidence cell' role='cell'>
-            {(result as MannWhitneyResultsItem).cles?.p_value_cles || '-'}
-          </div>
-        ) : (
-          <div className='confidence cell' role='cell'>
-            {confidenceText && confidenceIcons[confidenceText]}
-            {confidenceText || '-'}
-          </div>
-        )}
+        {renderDifferingTestVersionColumns(testVersion ?? STUDENT_T, result)}
```

Comment:
Thanks for making this change! Can you file a follow-up task to move the other test type specific code to here later? (I'm referring to lines 370-416)